### PR TITLE
Add reverse lift syntax to Storm (SYN-5290, SYN-5833)

### PR DESCRIPTION
**Pull request by Cisphyx**

### CODE REVIEW

File: synapse/lib/ast.py [hunk 1]
```diff
@@ -1202,6 +1202,14 @@ class CaseEntry(AstNode):
 
 class LiftOper(Oper):
 
+    def __init__(self, astinfo, kids=()):
+        Oper.__init__(self, astinfo, kids=kids)
+        self.reverse = False
+
+    def reverseLift(self, astinfo):
```
Comment (vEpiphyte) [inline synapse/lib/ast.py:1209]: coverage?

File: synapse/lib/parser.py [hunk 2]
```diff
@@ -445,6 +446,13 @@ def switchcase(self, meta, kids):
 
         return s_ast.SwitchCase(astinfo, newkids)
 
+    @lark.v_args(meta=True)
+    def liftreverse(self, meta, kids):
+        assert len(kids) == 1
+        astinfo = self.metaToAstInfo(meta)
+        kids[0].reverseLift(astinfo)
```
Comment (vEpiphyte) [inline synapse/lib/parser.py:453]: reverseLift

File: synapse/lib/ast.py (hunk 1 above)
Comment (Cisphyx) [reply synapse/lib/ast.py:1209]: cleaned up

File: synapse/lib/parser.py (hunk 2 above)
Comment (Cisphyx) [reply synapse/lib/parser.py:453]: coverage cleaned up

**Review by invisig0th — APPROVED** [review]

**Review by vEpiphyte — APPROVED** [review]

**Pull request merged**